core/whistle_apps-1.0.0/src/whistle_apps_init.erl

```diff
@@ -9,17 +9,20 @@
 -module(whistle_apps_init).
 
 -export([start_link/0
+         ,sanity_checks/0
          ,init/0
         ]).
 
 -include("whistle_apps.hrl").
 
 start_link() ->
+    _ = sanity_checks(), %% one day make this true
```

Comment:
Well, the debate was whether to crash the process (and ultimately the app/VM) if the sanity checks didn't hold. For now, we just log critically but we could make this `'true' = sanity_checks()` to really enforce the fixing of these checks.